sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -348,6 +357,7 @@ def list_deleted_keys(self, **kwargs):
         pages = self._client.get_deleted_keys(self._vault_url, maxresults=max_page_size, **kwargs)
         return (DeletedKey._from_deleted_key_item(item) for item in pages)
 
+    @distributed_trace
```

Comment:
Can you please check what the resulting spans are from the `list_<something>` methods. The methods themselves should be returning `Paged[<something>]` instances that, in turn, will make individual service calls on demand if/when the caller iterates through the list.

Reply:
yes that is what happens. 😄 

![image](https://user-images.githubusercontent.com/13604256/61556128-e684aa80-aa15-11e9-9bdd-0b4cfcc45742.png)